documentstore_migracao/export/sps_package.py

```diff
@@ -556,6 +556,36 @@ def transform_article_meta_count(self):
 
         return self.xmltree
 
+    def remove_article_body_when_text_is_available_only_in_pdf(self) -> None:
+        """Remove o corpo do documento quando o texto completo está disponível
+        apenas em pdf."""
+
+        def find_pdf_links(body: etree.Element) -> list:
+            TEXT_LINK_FOR_PDF = [
```

Comment:
Nós temos segurança de que há apenas estas 3 variações? Pergunto isso pois o texto é inserido manualmente.

Reply:
Sem avaliar todos os htmls eu não tenho como afirmar isso @gustavofonseca, porém observando alguns volumes da revista `mioc`, eu vi artigos com pdfs nos idiomas `pt`, `en`, `fr`, `es` e `de`, e que no texto os links estavam apenas em `en` e `es`.

1) [pt](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0074-02761917000100005&lng=en&nrm=iso&tlng=pt)
2) [pt-en](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0074-02761918000100005&lng=en&nrm=iso&tlng=pt)
3) [pt-de](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0074-02761924000100003&lng=en&nrm=iso&tlng=pt)
4) [fr-pt](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0074-02761922000200010&lng=en&nrm=iso&tlng=pt)
5) [es](http://www.scielo.br/scielo.php?script=sci_arttext&pid=S0074-02761929000200002&lng=en&nrm=iso&tlng=es)

Reply:
Vou colocar de outro jeito: é seguro dizer que este _match_ `node.text.lower() in TEXT_LINK_FOR_PDF` ocorrerá em todos os casos desejados?

Reply:
> Vou colocar de outro jeito: é seguro dizer que este _match_ `node.text.lower() in TEXT_LINK_FOR_PDF` ocorrerá em todos os casos desejados?

Se não me falhe a memória, esses textos são colocados pelo site clássico.

@joffilyfe, seria bom ter certeza disso porque a tradução em espanhol que encontrei no https://github.com/scieloorg/Web não tem o ponto final (`.`) na frase. Você tem algum exemplo do texto em algum artigo que está em espanhol?

Reply:
@patymori alterei o código para ser compatível com a [descoberta](https://github.com/scieloorg/document-store-migracao/pull/249#issuecomment-579349589) do @gustavofonseca, as alterações estão em c1eeeeeedb437041268e6c95011135420a76d2e1.

Reply:
@gustavofonseca 

Bem observado!